### PR TITLE
Fixed linear depreciation on annual period for fixed assets

### LINKED ISSUE #1977: Annual depreciation are not working anymore
The annual period and linear depreciation option doesn't create as many depreciations as wanted.

immo_1

immo_2

1- How to reproduce?

Input a fixed asset with 50 000 €, linear option and annual period. started_on 2017-01-01 and stopped-on 2020-12-31

2- What is really expected?

Having 3 depreciation instead of 1.

3- How to test it?

On https://demo.ekylibre.farm/backend/fixed-assets

### PR DESCRIPTION
**Pull request by Zank94**
Fixes #1977

### CODE REVIEW

**Comment by Zank94** [comment]
[ekylibre:debug/gh#1977-fix_linear_depreciation_on_annual_period](https://trello.com/c/RLebGQQs/259-ekylibredebug-gh1977-fixlineardepreciationonannualperiod)

File: .gitignore [hunk 1]
```diff
@@ -93,4 +93,5 @@ Procfile.*
 .gitconfig
 
 custom_plan.rb
-zeus.json
\ No newline at end of file
+zeus.json
+postgres_config
```
Comment (burisu) [inline .gitignore]: Not the subject of PR

File: postegres_config [hunk 2]
```diff
@@ -0,0 +1,623 @@
+# -----------------------------
```
Comment (burisu) [inline postegres_config]: This file should not be in PR

File: .gitignore [hunk 3]
```diff
@@ -93,4 +93,4 @@ Procfile.*
 .gitconfig
 
 custom_plan.rb
-zeus.json
\ No newline at end of file
+zeus.json
```
Comment (burisu) [inline .gitignore]: Not concerned by PR, restoration needed.

**Pull request merged**